lib/rules/jest/no-vague-titles.js

```diff
@@ -103,7 +131,9 @@ function getDescription({arguments: args}) {
   return firstArgument && firstArgument.value;
 }
 
-function containsVagueWord(description) {
-  const vagueTerms = [/correct/i, /appropriate/i, /\ball\b/i];
-  return vagueTerms.find((term) => description.match(term));
+function containsVagueWord(description, allowed) {
+  return Object.entries(VAGUE_TERMS).find(
```

Comment:
Neat!  Could you write a test that verifies that allowing one term doesn't let other slip through, please.  e.g., this should fail on "properly":

```
{code: "it('properly all'), options: [{allow: ['all']}]}
```